internal/server/kong/ws/config/version_compatibility.go

```diff
@@ -263,6 +263,44 @@ func (vc *WSVersionCompatibility) processConfigTableUpdates(uncompressedPayload
 	return processedPayload, nil
 }
 
+func (vc *WSVersionCompatibility) removeCoreEntity(
+	processedPayload string,
+	entityType string,
+	dataPlaneVersionStr string,
+	configTableKey string,
+	changeID ChangeID,
+	tracker *ChangeTracker,
+) string {
+	entity := gjson.Get(processedPayload, fmt.Sprintf("config_table.%s", configTableKey))
```

Comment:
I would just fetch the ID as that's all we need.
```go
entityIDs := gjson.Get(processedPayload, fmt.Sprintf("config_table.%s.#.id", configTableKey))
```

Reply:
fixed this.